SortVisualization/SortVisualization/Models/ArrayTypeSort.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  ArrayTypeSort.swift
+//  SortVisualization
+//
+//  Created by Oleg Yankiwskyi on 5/7/18.
+//  Copyright © 2018 Oleg Yankiwskyi. All rights reserved.
+//
+
+import Foundation
+
+class ArrayTypeSort: ArrayTypeSortProtocol {
+
+    private let arrayTypeSort = [
+                        TypeSortEntity(title: "Insert", typeSort: TypeSortEnum.insert),
+                        TypeSortEntity(title: "Bubble", typeSort: TypeSortEnum.bubble),
+                        TypeSortEntity(title: "Selection", typeSort: TypeSortEnum.selection)
```

Comment:
`typeSort` type defined in `TypeSortEntity`, so no necessity to add `TypeSortEnum` - `.insert` is enough